app/vmui: fix behavior when changing url in global settings (#4332)
VictoriaMetrics/VictoriaMetrics: docs/CHANGELOG.md

## tip

> * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix behavior when changing url in global settings if it includes tenants. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4322).

Reviewer: ```suggestion
* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix bug with custom URL in global settings not respecting tenantID change. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4322).
```
